test.js

```diff
@@ -14,9 +14,88 @@ test('all errors are instances of LevelUPError', function (t) {
   t.end()
 })
 
+test('error with message has expected properties', function (t) {
+  const error = new errors.ReadError('foo')
+
+  t.is(error.type, 'ReadError')
+  t.is(error.name, 'ReadError')
+  t.is(error.cause, undefined)
+  t.is(error.message, 'foo')
+  t.end()
+})
+
+test('error without message has expected properties', function (t) {
+  const error = new errors.ReadError()
+
+  t.is(error.type, 'ReadError')
+  t.is(error.name, 'ReadError')
+  t.is(error.cause, undefined)
+  t.is(error.message, '')
+  t.end()
+})
+
+test('error with cause has expected properties', function (t) {
+  const cause = new Error('foo')
+  const error = new errors.ReadError(cause)
+
+  t.is(error.type, 'ReadError')
+  t.is(error.name, 'ReadError')
+  t.ok(error.cause === cause)
```

Comment:
```suggestion
  t.strictEqual(error.cause, cause)
```

Reply:
I felt `===` was more expressive, but no strong opinion. Seeing as `strictEqual` is an alias for `is` though, we might as well use `t.is(error.cause, cause)`?

Reply:
If `is` is the current way, then I'm +1 for it. To me this is more expressive (you can just scan the `t.*` method names to see what's going on), and the test framework can be more helpful, because it will show you the values if they differ